src/components/calcite-card/calcite-card.tsx

```diff
@@ -0,0 +1,193 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  Host,
+  Prop
+} from "@stencil/core";
+import { CSS, SLOTS } from "./resources";
+import { ENTER, SPACE } from "../../utils/keys";
+import { getElementDir } from "../../utils/dom";
+import { VNode } from "@stencil/core/dist/declarations";
+
+/**
+ * @slot thumbnail - [Required] A slot for adding a thumnail to the card.
+ * @slot header - [Required] A slot for adding a heading and an icon to the card.
+ * @slot - A slot for adding subheader/description content.
+ * @slot action - A slot for adding a single action as a button.
```

Comment:
Seems like this slot is called `button-action` but `action` here? Can we make sure these are in sync? Maybe we could just call this `primary-button`? or just have the user use the default slot for this? Do we need it?

Reply:
Remnant from before, will remove / update.